src/GraphRequest.ts

```diff
@@ -302,6 +303,29 @@ export class GraphRequest {
 		}
 	}
 
+	/**
+	 * @private
+	 * Checks if the content-type is present in the _headers property. If not present, defaults the content-type to application/json
+	 * @param none
+	 * @returns nothing
+	 */
+	private setHeaderContentType(): void {
+		if (this._headers === undefined || this._headers === null) {
+			this.header("Content-Type", "application/json");
+		}
```

Comment:
```suggestion
			return;
		}
```

Reply:
Curious to know why an empty return is better than no return @zengin

Reply:
If `headers` is `undefined` or `null`, current code will set the header and continue to the logic that follows, which attempts to iterate over `headers` to search for `content-type`.

Reply:
Interesting, this makes lines 316 to 326 not necessary, right?

Since by line 316 the content-type property will always be present.

Reply:
Not for all cases, that's why we need a return to stop execution if the headers are null or undefined. If they are defined, we can still proceed with the logic to check if "content-type" header exists.

More reading on this topic:
1. Arrowhead anti pattern and early return as a solution: https://stackoverflow.com/questions/17804005/how-to-prevent-the-arrowhead-anti-pattern
2. Early return vs single exit: https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement

Early returns are a widely adapted style in these situations. Example:
```
if (someObject == null)
{
  return false;
}

// someObject can safely be assumed not null at this point:
var a = someObject.SomeProperty;
```